packages/web/src/pages/upload-page/pages/SelectPage.tsx

```diff
@@ -35,7 +35,7 @@ export const SelectPage = (props: SelectPageProps) => {
   const onSelectTracks = useCallback(
     async (selectedFiles: File[]) => {
       const existing = new Set(
-        tracks.map(({ file }) => `${file.name}-${(file as File).lastModified}`)
+        tracks.map(({ file }) => `${file!.name}-${(file as File).lastModified}`)
```

Comment:
this is required bc `file` could be optional now? maybe leave a comment explaining why it's guaranteed to exist here?